content/en/apps/guides/hosting/app-developer.md

```diff
@@ -68,7 +70,10 @@ CHT_HTTPS=8443
 Now it's easy to boot this environment by specifying which `.env` file to use:
 
 ```shell
-docker-compose --env-file ../the_second/.env-docker-compose -f docker-compose-developer.yml up
+docker-compose --env-file ../the_second/.env-docker-compose -f docker-compose-developer-3.x-only.yml
+
+
+up
```

Comment:
```suggestion
docker-compose --env-file ../the_second/.env-docker-compose -f docker-compose-developer-3.x-only.yml up
```

Reply:
oh! typo that would cause an error - nice catch!